components/landing/LandingCta.vue

```diff
@@ -1,16 +1,23 @@
 <template>
-  <a class="landing-cta" target="_blank" rel="noopener">
+  <a
```

Comment:
Just a comment. We should replace the `a` component with `AppLink` when merging [this](https://github.com/Qiskit/qiskit.org/pull/592)